src/components/TasksSubList/__tests__/TasksSubList.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { tasksResponse } from '__mocks__';
+import TasksSubList from '../';
+
+jest.mock('@material-ui/core/Box', () => 'Box');
+jest.mock('@material-ui/core/Typography', () => 'Typography');
+jest.mock('components/TaskCheckbox', () => 'TaskCheckbox');
+
+describe('Component - TasksSubList', () => {
+  test(`renders correctly Todo section`, () => {
+    const component = create(
+      <TasksSubList title='Todo' tasks={tasksResponse('uncompleted')} count={1} />
```

Comment:
I think here you can leverage  `TaskStatus` enum and not passing strings `'uncompleted'` / `'completed'`.
```import { TaskStatus } from '__mocks__';```

Same occurs with function `tasksResponse` inside `__mocks__/graphql`

```
  switch (filter) {
    case 'completed':
      return [tasks[0]];
    case 'uncompleted':
      return [tasks[1]];
    default:
      return tasks;
  }
```

Although this is test file, so it's not a bloker in any means :+1: